pkg/csi/plugins/nodeserver.go

```diff
@@ -167,6 +172,23 @@ func (ns *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublis
 func (ns *nodeServer) NodeUnpublishVolume(ctx context.Context, req *csi.NodeUnpublishVolumeRequest) (*csi.NodeUnpublishVolumeResponse, error) {
 	targetPath := req.GetTargetPath()
 
+	// check path existence
+	_, err := os.Stat(targetPath)
```

Comment:
Also need to check target is valid or not(0 length).

Reply:
ok